addons/l10n_ph/data/account_fiscal_position_template.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<odoo>
+    <record id="fiscal_position_template_vat_registered" model="account.fiscal.position.template">
+        <field name="sequence">1</field>
+        <field name="name">VAT Registered</field>
+        <field name="chart_template_id" ref="l10n_ph_chart_template" />
+        <field name="auto_apply" eval="True" />
+        <field name="vat_required" eval="True" />
+        <field name="country_id" ref="base.ph" />
+    </record>
+    <record id="fiscal_position_template_non_vat" model="account.fiscal.position.template">
+        <field name="sequence">2</field>
+        <field name="name">Non-VAT</field>
+        <field name="chart_template_id" ref="l10n_ph_chart_template" />
+        <field name="auto_apply" eval="True" />
+        <field name="country_id" ref="base.ph" />
+    </record>
+    <record id="fiscal_position_template_vat_exempt" model="account.fiscal.position.template">
+        <field name="sequence">3</field>
+        <field name="name">VAT Exempt</field>
+        <field name="chart_template_id" ref="l10n_ph_chart_template" />
+        <field name="auto_apply" eval="True" />
+        <field name="country_id" ref="base.ph" />
+    </record>
+    <record id="fiscal_position_template_zero_rated" model="account.fiscal.position.template">
+        <field name="sequence">4</field>
+        <field name="name">Zero-Rated</field>
+        <field name="chart_template_id" ref="l10n_ph_chart_template" />
+        <field name="auto_apply" eval="True" />
+        <field name="country_id" ref="base.ph" />
```

Comment:
I don't see any difference between non-vat, vat exempt, and zero-rated fiscal positions conditions of application, which means that the last 2 won't be used. Do I miss something? why are they there?

Also the zero-rated, vat exempt, and non-vat taxes have the exact same effect on journal entries (and there is no vat report), so again these seem useless?

Reply:
Its from the government, we still need to include one of them, we will include vat exempt only.